src/Compilers/Core/Portable/Compilation/ParseOptions.cs

```diff
@@ -32,9 +32,13 @@ public abstract class ParseOptions
         public SourceCodeKind SpecifiedKind { get; protected set; }
 
         /// <summary>
-        /// Gets a value indicating whether the documentation comments are parsed.
+        /// Gets a value indicating whether the documentation comments are parsed and can produce diagnostics, parsed only, or treated as regular comments.
         /// </summary>
-        /// <value><c>true</c> if documentation comments are parsed, <c>false</c> otherwise.</value>
+        /// <value>
+        /// <para><see cref="DocumentationMode.Diagnose"/> if documentation comments are parsed and can produce diagnostics.</para>
+        /// <para><see cref="DocumentationMode.Parse"/> if documentation comments are parsed without producing diagnostics.</para>
```

Comment:
Consider saying: "if documentation comments are parsed, but not analyzed." I am assuming parsing itself can cause some diagnostics to be reported. So, "without producing diagnostics" is probably not accurate.

Reply:
The doc for `DocumentationMode.Parse` says: "Parses documentation comments as structured trivia, but do not report any diagnostics.". I'll try to check if parsing itself can produce diagnostics or not, and update both comments.

Reply:
If enum values are documented, then why duplicate the information here at all? This is going to create an unnecessary maintenance burden, in my opinion.

Reply:
Sounds reasonable. I'm going to delete the "value" tag.